addons/rose/addon/helpers/map-type-to-color.js

```diff
@@ -0,0 +1,21 @@
+/**
```

Comment:
this is a simple helper made to map the flash message type to hds toast color. Let me know if y'all have a better suggestion.